ds_test.go

```diff
@@ -546,7 +546,7 @@ func TestDiskUsage(t *testing.T) {
 		t.Fatal(err)
 	}
 	s, _ := d.DiskUsage()
-	if s <= 0 {
+	if s == 0 {
```

Comment:
Why did we need to change this?

Reply:
I believe that was your change:
> fix lint error
https://github.com/ipfs/go-ds-badger2/pull/27/commits/1b16498077eff4df0079eb985e932d0a24aee342

Reply:
Ah... because disk usage is an unsigned int. Nevermind...